### PR TITLE
docs: Update development.md

### PR DESCRIPTION
**Pull request by richb-hanover**
Second attempt to create a PR - this is patch-3

I don't know what happened to the previous PR - It never appeared in the PRQL repo, nor could I push these changes (using patch-2)

### CODE REVIEW

File: web/book/src/project/contributing/development.md [hunk 1]
```diff
@@ -435,37 +437,40 @@ task run-playground
 
 Currently we release in a semi-automated way:
 
-1. PR & merge an updated
-   [Changelog](https://github.com/PRQL/prql/blob/main/CHANGELOG.md). GitHub will
-   produce a draft version at <https://github.com/PRQL/prql/releases/new>,
-   including "New Contributors".
+- [ ] PR & merge an updated
```
Comment (max-sixty) [inline web/book/src/project/contributing/development.md]: Do you know whether these render reasonably on the mdbook?

File: web/book/src/project/contributing/development.md [hunk 2]
```diff
@@ -435,37 +437,40 @@ task run-playground
...
-4. From there, both the tag and release is created and all packages are
-   published automatically based on our
-   [release workflow](https://github.com/PRQL/prql/blob/main/.github/workflows/release.yaml).
-5. Run
-   `cargo release version patch -x --no-confirm && cargo release replace -x --no-confirm`
-   to bump the versions and add a new Changelog section; then PR the resulting
-   commit.
-6. Check whether there are [milestones](https://github.com/PRQL/prql/milestones)
-   that need to be pushed out.
+- [ ] Review the **Current Status** on the README.md to ensure it reflects the
```
Comment (max-sixty) [inline web/book/src/project/contributing/development.md]: Maybe at the end? It doesn't need to block the other things but I agree it's a good time to update it (your last update was great!)

File: web/book/src/project/contributing/development.md [hunk 3]
```diff
@@ -88,12 +88,14 @@ since it relies on `brew`.
...
 
-### Option 3: Use a [Dev Container](https://containers.dev/)
+### Option 3: Use a Dev Container
 
 This project has a
 [devcontainer.json file](https://github.com/PRQL/prql/blob/main/.devcontainer/devcontainer.json)
 and a
 [pre-built dev container base Docker image](https://github.com/PRQL/prql/pkgs/container/prql-devcontainer-base).
+Dev Containers are specified at
+[https://containers.dev/](https://containers.dev/)
```
Comment (max-sixty) [inline web/book/src/project/contributing/development.md]: Link is great but maybe we don't need this now you added it to the title?

Comment (richb-hanover) [reply web/book/src/project/contributing/development.md]: Actually, I took the link _out_ of the title/heading because there was no link icon when I viewed the Markdown. (Those link icons were present for Options 1, 2, and 4)

But when looking at the Book directly either one looks good. Let's leave this like this.

File: web/book/src/project/contributing/development.md (hunk 2 above)
Comment (richb-hanover) [reply web/book/src/project/contributing/development.md]: OK. Done. But see #3778

File: web/book/src/project/contributing/development.md [hunk 4]
```diff
@@ -435,37 +437,44 @@ task run-playground
 
 Currently we release in a semi-automated way:
 
-1. PR & merge an updated
-   [Changelog](https://github.com/PRQL/prql/blob/main/CHANGELOG.md). GitHub will
-   produce a draft version at <https://github.com/PRQL/prql/releases/new>,
-   including "New Contributors".
+- PR & merge an updated
```
Comment (max-sixty) [inline web/book/src/project/contributing/development.md]: Why the change from numbered? I obv don't have a super-strong view, but I had thought that making them numbered showed they were ordered?

Comment (richb-hanover) [reply web/book/src/project/contributing/development.md]: Good point. Updated.

File: web/book/src/project/contributing/development.md [hunk 5]
```diff
@@ -451,21 +453,28 @@ Currently we release in a semi-automated way:
...
-   maintainers have access to this page.}}, copy the changelog entry into the release
-   description{{footnote: Unfortunately GitHub's markdown parser
-    interprets linebreaks as newlines. I haven't found a better way of editing
-    the markdown to look reasonable than manually editing the text.}}, enter the
-   tag to be created, and hit "Publish".
+       maintainers have access to this page.}}, copy the changelog entry into the
+   release description{{footnote: Unfortunately GitHub's markdown parser
+        interprets linebreaks as newlines. I haven't found a better way of editing
+        the markdown to look reasonable than manually editing the text.}}, enter
+   the tag to be created, and hit "Publish".
```
Comment (max-sixty) [inline web/book/src/project/contributing/development.md:463]: Some odd indents here...

Comment (richb-hanover) [reply web/book/src/project/contributing/development.md:463]: I'm pretty sure this PR addresses the odd indents that are currently in the Book (https://prql-lang.org/book/project/contributing/development.html)

The problem is that the `admonish note` cannot be indented: it gets screwed up when there are spaces at the front. But maybe I'm not seeing what you noticed

Comment (max-sixty) [reply web/book/src/project/contributing/development.md:463]: OK cool, it does look good in HTML!

**Comment by max-sixty** [comment]
Thanks @richb-hanover !
